### DISCUSSION TITLE
Airflow executor parallel or sequential

### DISCUSSION THREAD

**Discussion by tkranenburg** [apache/airflow#25470] [Q&A] [answered]
Dear all,

It is my first post here, and I have a question.
Within the airflow configurations, the default executor is sequential (executor = SequentialExecutor). 
When DAGs should run in parallel, it can be changed to: executor = LocalExecutor.

I am wondering if it is possible to indicate that some DAGs should only be run sequentially, while others can be run in parallel? The reason being that some jobs require a license that cannot be ran in parallel.

Looking forward to your replies, and if you have any suggestions where to start with.

Kind regards,

**Comment by potiuk** [comment, marked as answer]
There are various ways you can achieve that - for example you can have a pool with size 1 which is assigned to all tasks that need to be run sequentially or you can add some parameters to dag to make sure that only one dag run - just look through https://airflow.apache.org/docs/apache-airflow/2.3.3/concepts/index.html  and DAG object API.

**Reply by tkranenburg** [reply]
Thanks for pointing me in the right direction. That will help for sure!
